tests/fixtures/MakeFormForEntity/src/Entity/SourFood.php

```diff
@@ -23,7 +23,7 @@ class SourFood
     private $title;
 
     /**
-     * @ORM\OneToMany(targetEntity="App\Entity\Property", mappedBy="sourFood")
+     * @ORM\OneToMany(targetEntity=Property::class, mappedBy="sourFood")
      */
     private $properties;
 
```

Comment:
Could we add a test case for when the 2 entities are *not* in the same namespace?

Reply:
In this case, add another relationship with `Property` from another namespace?

- App\Entity\SourFood
- App\Entity\Property
- Foo\Bar\Property

Reply:
That's exactly what I was thinking. Or, I think it would be fine to create another entity in a sub-directory - e.g. `App\Entity\SubDirectory\SomeOtherClass` - and them make a relationship from `SourFood` to *that* entity. That would still require a `use` statement (so it would be a good example) but having it inside of the `Entity/` directory might be easier to get working :)

Reply:
Added test to add new relation with same short class name: https://github.com/symfony/maker-bundle/pull/573/commits/5f2a8b968a6368af66f006572f4a3c5a40744f4d#diff-55d2be7d85da27a6541ac1a80e425d1c